### DISCUSSION TITLE
method for outputting parameters

### DISCUSSION THREAD

**Discussion by afriedman412** [MaartenGr/BERTopic#1122] [Ideas]
I'm doing some hyperparameter tuning and

it would be great if there were a clean way to output all the parameters for a run

like to a JSON or whatever

**Comment by MaartenGr** [comment]
You can use `.get_params()` to extract all parameters from BERTopic. You can then parse it to the structure that fits best with your use case.

**Comment by afriedman412** [comment]
great!
